plugins/font/plugin.js

```diff
@@ -1,45 +1,75 @@
-﻿/**
+/**
  * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
  * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
  */
 
 ( function() {
-	function addCombo( editor, comboName, styleType, lang, entries, defaultLabel, styleDefinition, order ) {
+	function addCombo( editor, comboName, styleType, lang, entries, defaultLabel, configStyleDefinition, order ) {
 		var config = editor.config,
-			style = new CKEDITOR.style( styleDefinition );
+			allowedAndRequiredContent = new CKEDITOR.style( configStyleDefinition ),
```

Comment:
Maybe something like `contentStyle` or `defaultContentStyle` as it's also used in refresh methods?